packages/react-query/src/__tests__/useQuery.test.tsx

```diff
@@ -862,71 +857,6 @@ describe('useQuery', () => {
     })
   })
 
-  it('should not get into an infinite loop when removing a query with cacheTime 0 and rerendering', async () => {
```

Comment:
can we keep the test but implement the removing with `queryClient.removeQueries({ queryKey: key })` instead? I think the test itself is still valuable

Reply:
Done ✅